.circleci/config.yml

```diff
@@ -64,7 +65,8 @@ references:
     DO_TF_WORKSPACE: dev
     RELEASE_NAME: codacy-hourly
     NAMESPACE: codacy-hourly
-    K8S_VERSION: "1.14"
+    ENVIRONMENT: sandbox
```

Comment:
Used for pattern-matching with the templated values-db-<environment>.yaml file created by terraform